src/ui_manager.h

```diff
@@ -97,7 +98,7 @@ class ui_adaptor
         explicit ui_adaptor( debug_message_ui );
         ui_adaptor( const ui_adaptor &rhs ) = delete;
         ui_adaptor( ui_adaptor &&rhs ) = delete;
-        ~ui_adaptor();
+        virtual ~ui_adaptor();
```

Comment:
I think this is no longer necessary now that `cataimgui::window_impl` uses a `ui_adaptor` member instead of inheriting from `ui_adaptor`.